OJP/OJP_JourneySupport.xsd

```diff
@@ -503,6 +503,25 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:simpleType name="UseRealtimeDataEnumeration">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="full">
+				<xs:annotation>
+					<xs:documentation>Full usage of realtime information, including the removing of SERVICE JOURNEYS.</xs:documentation>
```

Comment:
`<xs:documentation>Full use of real-time information, including removal of SERVICE JOURNEYS.</xs:documentation>`

Reply:
changed